bot/cogs/rss.py

```diff
@@ -0,0 +1,94 @@
+from asyncio import sleep
+from datetime import timedelta
+from typing import Dict, List
+
+import arrow
+import discord
+import feedparser
+
+from discord.ext.commands import Bot, Cog, Context, hybrid_command
+
+RPI_RSS_FEED_URL = "https://rpilocator.com/feed/"
+
+NOTIF_COOLDOWN = timedelta(hours=3)
+
+
+class NotifyMember:
+    def __init__(self, search: str, member: discord.Member, channel: discord.TextChannel):
+        self.search_term: str = search
+        self.member: discord.Member = member
+        self.notify_channel: discord.TextChannel = channel
+        self.notified: bool = False
+
+
+notifications: Dict[int, Dict[str, NotifyMember]] = {}
+
+
+def data_to_ping_from_rss(search_term: str, feed_data: List[dict]):
+    now = arrow.utcnow()
+    for entry in feed_data:
+        time_fmt = "ddd, DD MMM YYYY HH:mm:ss"  # Time format: Mon, 01 May 2023 15:19:40 GMT
+        published = arrow.get(entry["published"], time_fmt, tzinfo="UTC")
+
+        if search_term.lower() in entry["title"].lower() and now - published <= NOTIF_COOLDOWN:
+            return entry["title"]
+    return None
+
+
+class RSS(Cog):
+    """
+    RSS feed notifier.
+
+    Current implementation is a hacky solution to get Raspberry Pi notifications ASAP
+
+    This cog is not safe and needs to be modified before trusting arbitrary RSS feeds
+    """
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @Cog.listener()
+    async def on_ready(self):
+        while True:
+            rss_data = feedparser.parse(await self.pull_rss_feed(RPI_RSS_FEED_URL))["entries"][:10]
+
+            for user_id in notifications:
+                user_notifs = notifications[user_id]
+                for notif in user_notifs.values():
+                    ping_member_data = data_to_ping_from_rss(notif.search_term, rss_data)
+                    if ping_member_data and not notif.notified:
+                        notif.notified = True
+                        await notif.notify_channel.send(f"{notif.member.mention}--{ping_member_data}")
+            await sleep(10)
+
+    async def pull_rss_feed(self, url: str):
+        headers = {
+            "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) "
+                          "AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.246"
+        }
+        async with self.bot.session.get(url, headers=headers) as response:
+            return await response.text()
+
+    @hybrid_command(brief="Create or delete a new RSS notification", aliases=["create_notification", "notification"])
```

Comment:
Aliases are constants.

```suggestion
    @hybrid_command(brief="Create or delete a new RSS notification", aliases=("create_notification", "notification"))
```